### PR TITLE
Customize prettier for gql2ts generated .ts file

### LINKED ISSUE #3991: Fix the issue where packages/client/types/graphql.ts is changed and staged whenever server is started
## Issue
Every time we run `yarn dev`, `packages/client/types/graphql.ts` will be changed with `;` added on every single line. Therefore Git marked it as a changed file and put it in staging area. This makes committing and pushing cumbersome. 

### Acceptance Criteria (optional)
- [ ] `packages/client/types/graphql.ts` should not be in Git staging area if it's not changed explicitly by developer

### PR DESCRIPTION
**Pull request by tianrunhe**
This fixes #3991 .

@gql2ts/language-typescript has a customized [prettier](https://github.com/amount/gql2ts/blob/master/packages/language-typescript/src/typescriptPrettify.ts#L17-L24) that's different than our configurations. It's [this change](https://github.com/ParabolInc/parabol/commit/610a181a2cfacad1160e30649690448d854057ce#diff-b89bfe1397acd7451f86caf05f468827) that brings the issue. The fix is to overwrite the prettier configuration to match [ours](https://github.com/ParabolInc/parabol/blob/master/.prettierrc). So only intended change will be in the diff.

### **Tests:**

I made some random addition to the `NewMeeting` type:

then I run `yarn dev` as usual and observed the generated file of `client/types/graphql.ts`:

You can see only the intended diffs are there, as compared before you have diffs everywhere on semicolons.

### CODE REVIEW

**Comment by mattkrick** [comment]
this definitely solves it! unfortunately, `updateGQLSchema` runs every time the dev server starts up or the schema changes, which means it has to wait for the formatter to complete.

alternatively, since graphql.ts is autogenerated, i propose we just ignore it from getting formatted. that way it'll be as fast as possible.

**Comment by tianrunhe** [comment]
@mattkrick The only way I can make `lint-staged` to ignore the file is by explicitly not matching `graphql.ts` and also the match pattern has to include the base paths.

File: packages/client/lint-staged.config.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 module.exports = {
-  '**/*.{js,ts,tsx}': 'eslint --fix',
-  '**/*.{ts,tsx}': () => 'tsc --noEmit -p tsconfig.json'
+  'package/client/**/!(*graphql).{js,ts,tsx}': 'eslint --fix',
+  'package/client/**/!(*grphql).{ts,tsx}': () => 'tsc --noEmit -p tsconfig.json'
```
Comment (mattkrick) [inline packages/client/lint-staged.config.js:3]: i don't think the tsc needs the negation regex (it's also missing an `a`, so probably isn't doing much 😋 )

Comment (tianrunhe) [reply packages/client/lint-staged.config.js]: Oops! Fixed.

**Pull request merged**